app/actions/v3/service_instance_create_managed.rb

```diff
@@ -185,6 +189,14 @@ def parse_response(fetch_instance, last_operation)
         }
       end
 
+      def validate_service_instance!(instance)
+        if instance
+          instance_already_exists!(instance.name) if instance.create_succeeded? || instance.create_in_progress? || instance.update_succeeded? ||
```

Comment:
We basically raise an error when the last operation type is `update` - regardless of the state. It makes sense to enhance the `ServiceOperationMixin` with a method that answers this question. There already is a private method `update?`, so this one could be made public. But I would then also rename it, as I would not understand `instance.update?`. Maybe `instance.last_operation_update?`? Other ideas?

Reply:
Done. Called it `last_operation_is_update` otherwise it might be still confusing.